### ISSUE TITLE
mrr and linux

### ISSUE THREAD

**Issue by szentigrad3** [RainbowMiner/RainbowMiner#1180]
it is not creating the rigs on linux

**Comment by RainbowMiner** [comment]
Now that is interesting. It should, since there is no difference to Windows. Could you eventually upload a Debug file?

**Comment by szentigrad3** [comment]
[debug_2020-09-27.zip](https://github.com/RainbowMiner/RainbowMiner/files/5286777/debug_2020-09-27.zip)

**Comment by szentigrad3** [comment]
fixed it myself

**Comment by szentigrad3** [comment]
its only adding a limited amount of rigs any ideas why <> https://i.imgur.com/TwE4ooE.png

**Comment by RainbowMiner** [comment]
Yes - most probably there are profitability constrains in place.

Two possible factors:
- minimum profitability is not reached (pools.config.txt, parameter "AutoCreateMinProfitPercent")
- minimum hours to rent are above the maximum (pools.config.txt, parameter "AutoCreateMaxMinHours")
